package/gluon-alfred/Makefile

```diff
@@ -7,7 +7,7 @@ PKG_RELEASE:=1
 include ../gluon.mk
 
 define Package/gluon-alfred
-  DEPENDS:=+gluon-core +gluon-respondd +gluon-neighbour-info gluon-mesh-batman-adv +micrond +alfred
+  DEPENDS:=+gluon-core +gluon-respondd +gluon-neighbour-info gluon-mesh-batman-adv-common +micrond +alfred
```

Comment:
I think this dependency (and all similar) should be left unchanged - they actually depend on one of the batadv implementations, not just the common parts.